codegen/config.go

```diff
@@ -81,7 +83,7 @@ type TypeMapField struct {
 
 func (c *PackageConfig) normalize() error {
 	if c.Filename == "" {
-		return errors.New("Filename is required")
+		return nil
```

Comment:
Can we keep this for model and exec? maybe by just not calling normalize if !Resolver.IsDefined?

Reply:
Ah yes. my bad forgot it's shared! =x